### DISCUSSION TITLE
Using SSL to connect to postgres(AWS RDS) from Hasura

### DISCUSSION THREAD

**Discussion by IKI-Li** [hasura/graphql-engine#8437] [Question]
I want to set up a SSL connection between postgres and Hasura.I have some questions about SSL settings of Hasura.Could anyone offer some help.THX.

**Comment by thadguidry** [comment]
This seems useful to perhaps help:  https://hasura.io/docs/latest/databases/connect-db/cloud-databases/gcp/#step-7-optional-enable-ssl-connection
